platform/brains2/src/lib/sync.h

```diff
@@ -8,4 +8,5 @@ namespace Sync {
     uint32_t read();
     void write(uint8_t value);
     bool detect();
+    uint32_t syncPinState;
```

Comment:
I think this should be in the implementation file, unless we need global access to it for some reason.

Reply:
Yeah, I'll move it to the implementation file.